[F-4] html coding for remaining pages

nomarchia opened the pull request:
Added following new page templates:
Portfolio, Portfolio-newborns, Portfolio-babies;
Price;
Contact;
Data protection, Legal information;

Added multi-lang support for elements of pages;
Added StatisPageServlet for being able to access new pages;
Added these pages to excludedUrls in SecurityFilter;

nomarchia committed:
[f-4] Html coding for remaining pages
Added needed static images for portfolio pages;
Added data-protection.html and legal-information.html pages, that are share same css file;
Added multi-lang support for pages

nomarchia committed:
[f-4] Html coding for remaining pages
Added StaticPageServlet to process new pages (might remove it later);
Added actual links to pages to header nav menu and to footer;
Extracted page-main-title tag to common.css for reuse on several pages;
Added price page with multi-lang support;

nomarchia committed:
[f-4] Html coding for remaining pages
Small changes in common.css and price.html page;
Added portfolio general and newborns/babies pages;

nomarchia committed:
[f-4] Html coding for remaining pages
Added contact page with message form;

nomarchia committed:
[f-4] Html coding for remaining pages
Fixes to finish Contact form;
Added multi-lang support for all page elements;
Added changes to SecurityFilter (for being able to access static pages as GUEST);

Yuliia-Sh commented on src/main/webapp/assets/css/portfolio.css:
```diff
@@ -0,0 +1,63 @@
+    /*fix to remove space between footer and bg image: */
+    margin-bottom: 187px;
+}
+
```
Класс container-main не решает вопрос с футером?

nomarchia replied:
Это не с футером проблема, а с фоном - он сейчас взят от другой страницы. Это фикс чтобы не было зазора между фоном и футером, там оставалось пикселей 10 белой полосы. После того, как будут нормальные фоны от Лилит, это будет убрано (специально коммент оставил, чтобы самому потом не забыть).

Miari commented:
Please do the following changes to the html pages:
* Remove checkbox from the contact form
* Change "Login" to "Anmeldung" in header in german version
* Change "GE" to "DE" in language selection
* Change "Email" to "Емейл" in russion version on login page

nomarchia committed:
Merge branch 'develop' into f-4-html-coding-for-remaining-pages

nomarchia committed:
[f-4] Html coding for remaining pages
Added new backgrounds for new pages;
Small css changes;
Removed checkbox from contact form;

nomarchia committed:
Merge remote-tracking branch 'origin/f-4-html-coding-for-remaining-pages' into f-4-html-coding-for-remaining-pages

nomarchia committed:
[f-4] Html coding for remaining pages
Edited contact form to make it working;
Added ContactFormServlet;

Miari commented on src/main/java/com/photostudio/web/servlet/ContactFormServlet.java:
```diff
@@ -0,0 +1,79 @@
+        stringBuilder.append("Message: ");
+        stringBuilder.append(message);
+
+        mailSender.send("New message via contact form", stringBuilder.toString(), "nomarchia2@gmail.com");
```
Admin's email needed here

Yuliia-Sh replied:
есть метод sendToAdmin

nomarchia replied:
Это было временное решение для теста, сейчас там sendToAdmin()

Miari commented on src/main/webapp/WEB-INF/templates/contact_de.properties:
```diff
@@ -0,0 +1,11 @@
+boy.label=Junge
+girl.label=Mädchen
+twins.label=Zwillinge
+message.placeholder=Botschaft
```
Nachricht

nomarchia replied:
changed

Miari commented on src/main/webapp/WEB-INF/templates/contact_de.properties:
```diff
@@ -0,0 +1,11 @@
+twins.label=Zwillinge
+message.placeholder=Botschaft
+name.placeholder=Name
+email.placeholder=Email
```
E-Mail. Email в немецком зубная эмаль ))

и Nachricht вместо Botschaft

nomarchia replied:
changed

Miari commented on src/main/webapp/WEB-INF/templates/contact.properties:
```diff
@@ -0,0 +1,11 @@
+name.placeholder=Name
+email.placeholder=Email
+city.placeholder=City
+index.placeholder=Index
```
ZIP

nomarchia replied:
changed

Miari commented on src/main/webapp/WEB-INF/templates/contact_de.properties:
```diff
@@ -0,0 +1,11 @@
+name.placeholder=Name
+email.placeholder=Email
+city.placeholder=Stadt
+index.placeholder=Index
```
PLZ

nomarchia replied:
changed

Miari commented on src/main/webapp/WEB-INF/templates/contact_en_US.properties:
```diff
@@ -0,0 +1,11 @@
+name.placeholder=Name
+email.placeholder=Email
+city.placeholder=City
+index.placeholder=Index
```
ZIP

Miari commented on src/main/webapp/WEB-INF/templates/data-protection_en_US.properties:
```diff
@@ -0,0 +1 @@
+data.title=Data Protection
```
protection

Miari commented on src/main/webapp/WEB-INF/templates/data-protection.properties:
```diff
@@ -0,0 +1 @@
+data.title=Data Protection
```
protection

Miari commented on src/main/webapp/WEB-INF/templates/fragment/footer.html:
```diff
@@ -1,15 +1,19 @@
     <div>
-        data protection
+        <a href="data-protection">
+            data protection
```
Data

Miari commented on src/main/webapp/WEB-INF/templates/fragment/footer.html:
```diff
@@ -1,15 +1,19 @@
     <div>
-        legal information
+        <a href="legal-information">
+            legal information
```
Legal

Miari commented on src/main/webapp/WEB-INF/templates/legal-information_de.properties:
```diff
@@ -0,0 +1 @@
+legal.title=Rechtsinformation
```
Impressum

Miari commented on src/main/webapp/WEB-INF/templates/legal-information.properties:
```diff
@@ -0,0 +1 @@
+legal.title=Legal Information
```
Imprint

Miari commented on src/main/webapp/WEB-INF/templates/legal-information_ru.properties:
```diff
@@ -0,0 +1 @@
+legal.title=Правовая информация
```
Правовая информация

Miari commented on src/main/webapp/WEB-INF/templates/legal-information_en_US.properties:
```diff
@@ -0,0 +1 @@
+legal.title=Legal Information
```
Imprint

nomarchia replied:
changed

Miari commented on src/main/webapp/WEB-INF/templates/legal-information.html:
```diff
@@ -0,0 +1,41 @@
+        <div class="container">
+            <div class="info-text-block">
+                <div class="info-title" th:text="#{legal.title}">
+                    Legal Information
```
Imprint

Miari commented on src/main/webapp/WEB-INF/templates/legal-information.html:
```diff
@@ -0,0 +1,41 @@
+    <meta th:replace="fragment/head.html :: meta">
+    <link th:replace="fragment/head.html :: link">
+    <link rel="stylesheet" type="text/css" href="assets/css/data-protection-legal-information.css">
+    <title th:text="#{legal.title}">Legal Info</title>
```
Imprint

nomarchia replied:
changed

Miari commented on src/main/webapp/WEB-INF/templates/portfolio-babies_ru.properties:
```diff
@@ -0,0 +1,2 @@
+portfolio.babies.title=Младенцы
```
Малыши

nomarchia replied:
поправлено)

nomarchia committed:
[f-4] Html coding for remaining pages
Made contact.html to be actually working:
Changed form's html;
Added backend support in ContactFormServlet;

Miari commented on src/main/webapp/WEB-INF/templates/portfolio_ru.properties:
```diff
@@ -0,0 +1,3 @@
+portfolio.title=Портфолио
+newborns.title=Новорожденные
+babies.title=Младенцы
```
Малыши

nomarchia replied:
поправлено)

nomarchia committed:
[f-4] Html coding for remaining pages
Refactored doPost() method of ContactFormServlet;
Added multi-lang texts for price.html, data-protection.html, legal-information.html;
Added changes for De/Ru labels according to Maryna's comments;

nomarchia committed:
[f-4] Html coding for remaining pages
Refactored ContactFormServlet;

nomarchia committed:
[f-4] Html coding for remaining pages
Quickfix: changed 'Index' to 'ZIP', according to Maryna's comment;

nomarchia committed:
[f-4] Html coding for remaining pages
Small fixes according to Maryna's comments;

nomarchia committed:
[f-4] Html coding for remaining pages
Quickfix: removed unnecessary texts;
